### DISCUSSION TITLE
Grafana on signage displays like Samsung Tizen - suggestion

### DISCUSSION THREAD

**Discussion by ianpmurphy** [grafana/grafana#51499] [General]
Theres quite a lot of postings on the community forum and on github about different versions of grafana not working with browser X or device Y.

I played around with the code to get a samsung device to pass the initial browser checks but Grafana just fails to load. This will undoubtedly be due to some library or other which Grafana uses and which fails to load, generating an error. I spent quite some time digging but unfortunately it is going to require me to get up to speed on too many new technologies to make this worth while continuing. I found I was going down a rabbit hole trying to get the page to load and really wasn't advancing.

The requirements for display of dashboards are simpler than requirements for editing a dashboard (i would imagine as there simply isn't so much interaction), so it should be posible to support a subset of functionality just for display purposes.

Theres a lot of people out there would like to use grafana on large signage type screens but can't due to browser compatability restrictions.

I'd like to suggest some changes to facilitage that the community help with compatabilty checks. To this end there needs to be a separation of code within grafana which would allow display only devices to run. No interaction complications.

Test page could be added which loaded different elements of Grafana dependencies, running a list of checks to indicate what a browser/device will allow you to do with grafana or where an issue was. This info could be reported back and be used to make modifications. Not sure how feasable this is to do but if you supply a starting point it would allow people like me to spend some time digging through an issue and trying to find a solution, which is good for everyone. At the moment I just have the impression of being faced by a monolithic problem and limited time to dedicate to it.

**Comment by renatoornelas** [comment]
Every new major release i see the same issues of grafana not loading on smart TV browsers and eventually they start working again.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
